src/main/java/hudson/plugins/jira/JiraReleaseVersionUpdater.java

```diff
@@ -25,11 +25,13 @@
 
 	private String jiraProjectKey;
 	private String jiraRelease;
+	private String jiraDescription;
 
 	@DataBoundConstructor
-	public JiraReleaseVersionUpdater(String jiraProjectKey, String jiraRelease) {
+	public JiraReleaseVersionUpdater(String jiraProjectKey, String jiraRelease, String jiraDescription) {
```

Comment:
please always leave the previous constructor and marl it as deprecated (always backward compat)